common/helpers/subnetmap.go

```diff
@@ -13,6 +13,7 @@ import (
 
 	"github.com/gin-gonic/gin"
 	"github.com/kentik/patricia"
+	"github.com/kentik/patricia/generics_tree"
 	tree "github.com/kentik/patricia/generics_tree"
```

Comment:
Duplicate. Replace occurrences of `generics_tree` by `tree`.

Reply:
done (I let auto-import catch me :) )